internal/database/webhooks.go

```diff
@@ -118,10 +120,42 @@ func (s *webhookStore) GetByRandomID(ctx context.Context, id string) (*types.Web
 	panic("Implement this method")
 }
 
-// Delete the webhook
-func (s *webhookStore) Delete(ctx context.Context, id int32) error {
-	// TODO(sashaostrikov) implement this method
-	panic("implement this method")
+const webhookDeleteQueryFmtstr = `
+-- source: internal/database/webhooks.go:Delete
+DELETE FROM webhooks
+WHERE uuid = %s
+`
+
+// Delete the webhook. Error is returned when provided UUID is invalid, the
+// webhook is not found or something went wrong during an SQL query.
+func (s *webhookStore) Delete(ctx context.Context, id uuid.UUID) error {
+	q := sqlf.Sprintf(webhookDeleteQueryFmtstr, id)
+	result, err := s.ExecResult(ctx, q)
+	if err != nil {
+		return errors.Wrap(err, "running delete SQL query")
+	}
+	rowsAffected, err := result.RowsAffected()
+	if err != nil {
+		return errors.Wrap(err, "checking rows affected after deletion")
+	}
+
+	if rowsAffected == 0 {
+		return errors.Wrap(&WebhookNotFoundError{ID: id}, "failed to delete webhook")
+	}
+	return nil
+}
+
+// WebhookNotFoundError occurs when a webhook is not found.
+type WebhookNotFoundError struct {
+	ID uuid.UUID
+}
+
+func (w *WebhookNotFoundError) Error() string {
+	return fmt.Sprintf("webhook with UUID=%q not found", w.ID)
```

Comment:
Feel free to ignore, but since it's a UUID we don't need quoting around it and the less characters/noise in an error message the better, IMHO.

```suggestion
	return fmt.Sprintf("webhook with UUID %s not found", w.ID)
```